src/chapters/setup-intro-data-types/why-java.rst

```diff
@@ -1,2 +1,21 @@
-Why Learn Another Language?
-============================
+Why Learn Java?
+===============
+
+This course is an introduction to Java and the Spring Boot framework. Java is a
+powerful programming language and one of the most widely used across the globe. 
```

Comment:
Sad face for Python!  Perhaps we will need to update this in a few years to avoid looking dated, but I suppose is can pass for now.

Depending on your web reference, JavaScript, Python, Java and some flavor of C all rank in the top 5 languages to know, with the order varying according to the author.